pkg/accounts/accountdb/dbo/user.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package dbo
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
+	"github.com/zeebo/errs"
+	"storj.io/storj/pkg/accounts/accountdb/dbx"
+	"time"
+)
+
+// User is a database object that describes User entity
+type User struct {
+	ID uuid.UUID
+
+	FirstName    string
+	LastName     string
+	Email        string
+	PasswordHash []byte
+
+	CreatedAt time.Time
+}
+
+// NewUser is a constructor for User entity
+func NewUser(id uuid.UUID, firstName, lastName, email string, password []byte, createdAt time.Time) *User {
```

Comment:
I think this constructor isn't helping you since writing the same code using `User{` is clearer.